src/app/GitCommands/Settings/AppSettings.cs

```diff
@@ -1993,6 +1998,8 @@ public static int RepoObjectsTreeStashesIndex
             set => SetInt("RepoObjectsTree.StashesIndex", value);
         }
 
+        public static ISetting<int> RepoObjectsTreeFavoritesIndex { get; set; } = Setting.Create(FavoriteIndex, nameof(RepoObjectsTreeFavoritesIndex), 5);
```

Comment:
Favorites should be first. Changing the default value of the other ROT root nodes made sense and worked. Please restore.
```suggestion
        public static ISetting<int> RepoObjectsTreeFavoritesIndex { get; set; } = Setting.Create(RepoObjectsTreePath, nameof(RepoObjectsTreeFavoritesIndex), 0);
```

Reply:
While the favorites index is currently set to 5, it can still be positioned as the first node in the tree and at the top of toggle buttons. Changing it to 0 was previously considered but raised concerns about disrupting earlier configurations. To preserve compatibility with existing setups, I suggest keeping the index at 5 rather than resetting it to 0.